src/command_modules/azure-cli-container/setup.py

```diff
@@ -14,7 +14,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "0.3.6"
+VERSION = "0.3.7"
```

Comment:
Update to 1.2.1 to fix a bug that prevents installation of your commands on MacOSX